From 1e1a7875cf6ff7a46e373ddb5c76621df72b8f74 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ville=20Syrj=C3=A4l=C3=A4?= <ville.syrjala@linux.intel.com>
Date: Thu, 16 Apr 2015 20:40:37 +0300
Subject: intel: Add marketing names for CHV
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

All CHV devices will be branded as "Intel(r) HD Graphics".

Signed-off-by: Ville Syrjälä <ville.syrjala@linux.intel.com>

diff --git a/src/intel_module.c b/src/intel_module.c
index bb74422..faf70ab 100644
--- a/src/intel_module.c
+++ b/src/intel_module.c
@@ -259,6 +259,12 @@ static const SymTabRec intel_chipsets[] = {
 	{0x163D, "HD graphics"},
 	{0x163E, "HD graphics"},
 
+	/* Cherryview (Cherrytrail/Braswell) */
+	{0x22b0, "HD Graphics"},
+	{0x22b1, "HD Graphics"},
+	{0x22b2, "HD Graphics"},
+	{0x22b3, "HD Graphics"},
+
 	/* When adding new identifiers, also update:
 	 * 1. intel_identify()
 	 * 2. man/intel.man
-- 
cgit v0.10.2

